src/components/Footer.tsx

```diff
@@ -82,20 +83,16 @@ export const Footer = (props: HTMLAttributes<HTMLDivElement>) => {
 		<div css={footerStyle} {...props}>
 			<div css={contentStyle}>
 				<div css={creditsStyle}>
-					<div>Created by <a href="https://topmonks.com">TopMonks</a></div>
+					<div>Created by <a href="https://taostats.io">TaoStats</a></div>
 					<div css={separatorStyle}></div>
-					<div>Powered by <a href="https://subsquid.io">Subsquid</a></div>
+					<div>Built on <a href="https://bittensor.com">Bittensor</a></div>
 				</div>
 				<div css={linksStyle}>
-					<a href="https://docs.calamar.app">Docs</a>
-					<a href="https://t.me/calamar_explorer">
-						<Telegram css={iconStyle} />
+					<a href="https://twitter.com/taostats">
+						<Twitter css={iconStyle} />
 					</a>
-					<a href="https://github.com/topmonks/calamar">
-						<GitHub css={iconStyle} />
-					</a>
-					<a href="mailto:calamar@topmonks.com">
-						<Email css={iconStyle} />
+					<a href="#">
```

Comment:
Discord link is: https://discord.com/channels/1086368192521318472/1086369375411507290